packages/cozy-harvest-lib/src/components/infos/TriggerErrorInfo.jsx

```diff
@@ -17,9 +18,21 @@ import Markdown from '../Markdown'
  * This component is strongly related with the content of locales files and
  * deals mainly with translation concerns.
  */
-export class TriggerErrorInfo extends PureComponent {
+export class TriggerErrorInfo extends Component {
```

Comment:
PureComponent has nothing to do with async stuff or else. It's the same thing than a component instead that it has a different logique on the shouldComponentUpdate. The component will not re-render itself if its props do not change. Thats it. 

Here I think we can keep the PureComponent, you can take a look for instance here https://github.com/cozy/orange-customizations/blob/61273d3c6e6498f55076afc72a9275b393423720/overrides/cozy-store/src/ducks/components/App.jsx